### PR TITLE
[#1008] download button fixed in course review of attchments (connects #1008)

### PR DESCRIPTION
**Pull request by razu9861**
screen shot 2017-09-14 at 10 50 24 am
screen shot 2017-09-14 at 10 56 45 am

### CODE REVIEW

File: app/MyApp/app/views/AnswerReview.js [hunk 1]
```diff
@@ -98,32 +100,20 @@ $(function () {
                 });
                 this.vars.questionlists.push(questionlist.toJSON()) 
                 this.vars.answerlist = this.collection.toJSON();
-                console.log(this.vars.answerlist)
                 var attchmentURL = null;
-                var attachmentName = null;
+                //var attachmentName = [];
```
Comment (nirojdyola) [inline app/MyApp/app/views/AnswerReview.js]: Do we need this ?

Comment (razu9861) [reply app/MyApp/app/views/AnswerReview.js]: removed

**Review by dogi — APPROVED** [review]
looks good so far

**Review by Rupesh87 — APPROVED** [review]

File: app/MyApp/app/views/listSyncDbView.js [hunk 2]
```diff
@@ -180,6 +206,7 @@ $(function() {
 				}
 			})
 		},
+
```
Comment (lmmrssa) [inline app/MyApp/app/views/listSyncDbView.js]: extra space

File: app/MyApp/index.html [hunk 3]
```diff
@@ -1700,10 +1700,14 @@ <h3 ><%=App.languageDict.attributes.Answer_Review_Section %></h3>
                             <td><%=App.languageDict.attributes.Question %></td>
                             <td id="markdown_question_<%=questionlists[i]._id%>"><textarea name="<%=questionlists[i]._id%>"><%=questionlists[i].Statement%></textarea></td>
                         </tr>
-                        <%if(attachmentName != null && attachmentName!= "" && attachmentName!= undefined ){%>
                         <tr>
-                            <td><a download="<%=attachmentName%>" href="<%=attchmentURL%>"  target="_blank" ><button class="btn btn-primary"><%=App.languageDict.attributes.Download %></button></a></td>
-                            <td></td>
+                            <td>Assignment Name</td>
```
Comment (lmmrssa) [inline app/MyApp/index.html]: Add translation

**Review by lmmrssa — CHANGES_REQUESTED** [review]

**Comment by razu9861** [comment]
@dogi Tests failing at Language selection, this PR has nothing to do with language selection

**Review by lmmrssa — APPROVED** [review]

**Pull request closed without merging**